### PR TITLE
Update for ghc-9

### PR DESCRIPTION
**Pull request by locallycompact**

### CODE REVIEW

File: src/Refined/Unsafe.hs [hunk 1]
```diff
@@ -28,6 +28,7 @@
 --------------------------------------------------------------------------------
 
 {-# LANGUAGE CPP #-}
+{-# LANGUAGE FlexibleContexts #-}
```
Comment (chessai) [inline src/Refined/Unsafe.hs:31]: why is this needed now?

Comment (locallycompact) [reply src/Refined/Unsafe.hs:31]: Due to this https://gitlab.haskell.org/ghc/ghc/-/wikis/migration/9.0#more-validity-checking-in-quantified-constraints

**Review by chessai — APPROVED** [review]

**Comment by locallycompact** [comment]
Could we get a release for ghc-9 with this please?
